Fix occ twofactor (#4209)
owncloud/docs: modules/user_manual/pages/personal_settings/security.adoc

= Security
:toc: right
:cors-url: https://en.wikipedia.org/wiki/Cross-origin_resource_sharing
:gps-ms-auth-url: https://play.google.com/store/apps/details?id=com.azure.authenticator&hl=en&gl=US
:ios-ms-auth-url: https://apps.apple.com/us/app/microsoft-authenticator/id983156458
:gps-google-auth-url: https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2&hl=en&gl=US
:ios-2fas-url: https://apps.apple.com/app/2fa-authenticator-2fas/id1217793794
:gps-2fas-url: https://play.google.com/store/apps/details?id=com.twofasapp&hl=en&gl=US

== Introduction

The security settings page allows for:

* Managing Two-Factor Authentication (if enabled by the administrator)
* Managing CORS white-listed domains
* Viewing and deleting user sessions
* Creating new app passcodes

== Settings

=== Two-Factor Authentication

> Two-Factor Authentication appears only, if enabled by the administrator. It is never enabled by default and as a user you can opt-in and also take back this kind af additional secrity measure. When enabled and setup, you have to enter additional to you password a 2nd factor which is a time-based one-time password (TOTP) provided by another device during the web-login. This device can be a mobile phone or another device where the 2nd factor is provided via e.g. an app or eg SMS.

Reviewer: ```suggestion
Two-Factor Authentication appears only if enabled by the administrator. It is never enabled by default and as a user you can opt-in and also stop this kind of additional security measure. When enabled and set up, you have to enter, in addition to your password, a second factor which is a time-based one-time password (TOTP) provided via another device during the web login. This device can be a mobile phone or another device providing the second factor, e.g. via an app or SMS.
```

> In the setup example below, an app on the mobile phone will be used providing the 2nd factor. For your convenience, some example apps are listed: 

Reviewer: ```suggestion
In the setup example below, an app on the mobile phone will be used to provide the second factor. For your convenience, some example apps are listed: 
```

iOS App Store::
{ios-ms-auth-url}[Microsoft Authenticator] +
{ios-2fas-url}[2FA Authenticator (2FAS)]

Google Play Store::
{gps-ms-auth-url}[Microsoft Authenticator] +
{gps-google-auth-url}[Google Authenticator] +
{gps-2fas-url}[2FA Authenticator (2FAS)]

.Enable Two-Factor Authentication
image::personal-settings/security/enable-2fa-totp.png[]

> After enabling Two-Factor Authentication via the btn:[Activate TOPT] checkmark, the image changes and you can proceed finalizing the setup.

Reviewer: ```suggestion
After enabling Two-Factor Authentication via the btn:[Activate TOPT] check mark, the image changes and you can proceed with finalizing the setup.
```

.Activated Two-Factor Authentication
image::personal-settings/security/activate-2fa-totp.png[]

> To finalize the setup, follow the apps way to scan the QR code and enter the number shown (2nd factor) in the browser at the btn:[Authentication code] field.

Reviewer: ```suggestion
To finalize the setup, follow the app's directions to scan the QR code. The number displayed (2nd factor) needs to be entered in the browser dialog in the btn:[Authentication code] field.
```

> Log out and on the next login, you will be asked past entering your user and password at a follow up screen to enter the number that the app provides. The number changes after some time and usually a counter is shown how much time is remaining after the next number appears.

Reviewer: ```suggestion
Log out and after your next regular login with user name and password, you will be asked to enter the number that the app provides. The number changes after some time and usually a counter is shown how much time is remaining after the next number appears.
```

==== Disable Two-Factor Authentication

You can disable the two-factor authentication at any time by unchecking btn:[Activate TOPT]

==== Lost or Defect Device

> If the device providing the second factor gets lost or is defect, the administrator can temporarily disable the two-factor authentication for the account in trouble. A login can then be made without providing the second factor. If the issue is solved, the administrator can reenable two-factor authentication.

Reviewer: ```suggestion
If the device providing the second factor gets lost or is defect, the administrator can temporarily disable the two-factor authentication for the account in question. A login can then be made without providing the second factor. If the issue is solved, the administrator can re-enable two-factor authentication.
```
